src/applications/accredited-representative-portal/accreditation/21a/pages/07-character-references-chapter/characterReferencesPages.js

```diff
@@ -36,12 +36,15 @@ const arrayBuilderOptions = {
     !item?.relationship,
   minItems: 3, // TODO: [Fix arrayBuilder minItems validation](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/87155)
   maxItems: 4,
+  showSummaryDescription: true,
   text: {
     getItemName: item =>
       `${item?.fullName?.first} ${item?.fullName?.last}${
         item?.fullName?.suffix ? `, ${item?.fullName?.suffix}` : ''
       }`,
     cardDescription: item => `${item?.phone}, ${item?.email}`,
+    summaryDescription:
+      'You must add at least 3 and no more than 4 character references.',
```

Comment:
Oh also for reference - William's PR is still in limbo related to min/max, since William transitioned to another team - https://github.com/department-of-veterans-affairs/vets-website/pull/31855 - Not sure who will be picking that back up - I can help out with that if needed eventually.

Reply:
That would be very helpful, as the entire ARF team will be ending after next sprint 🙈 !
This ticket is tangentially related to that work, and was raised by CAIA as part of their content review of the min/max validation.

Reply:
ok, I've added a ticket to our backlog, to help with that PR https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1656